### PR TITLE
Fixed site map button, added space before URL sequence

### PR DESCRIPTION
**Pull request by abhidas17695**

### CODE REVIEW

File: css/RTstyle.css [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 /* Modal Content */
 .modal-content {
+
```
Comment (vbanos) [inline css/RTstyle.css]: Why do you add empty space inside the CSS class definition?
In general there are many redundant spaces in this file which hinder readability.

File: css/sequences.css [hunk 2]
```diff
@@ -10,6 +10,7 @@
 /*  display: block;*/
   clear: both;
   padding-top: 10px;
+
```
Comment (vbanos) [inline css/sequences.css]: The same as with previous comment. Please remove the redundant spaces.

File: scripts/sequences.js [hunk 3]
```diff
@@ -295,6 +295,8 @@ chrome.runtime.onMessage.addListener(function(message, sender, sendResponse) {
 						var anc_arr = nodeArray;
 						// Data join; key function combines name and depth (= position in sequence).
 						var trail = document.getElementById("sequence");
+                        
+                        var textElem = document.createElement('span');
```
Comment (vbanos) [inline scripts/sequences.js]: You see how the identation is completely wrong here? Maybe it has something to do with your editor? Please fix it, all modifications in this file are totally off regarding identation.

**Comment by vbanos** [comment]
@abhidas17695 what about his PR?
It has fixes for sitemap button and other code for context menu (!)

I suggest to make another PR to fix the sitemap button and close this.

**Comment by abhidas17695** [comment]
The site map button has been fixed and the fix for the URLs was needed before the radial tree library was built. I will close this

**Pull request closed without merging**